src/params.cpp

```diff
@@ -1941,11 +1942,28 @@ bool ExecuteMulti(Cursor* cur, PyObject* pSql, PyObject* paramArrayObj)
                         bytes = PyUnicode_EncodeUTF16(source, cb, "strict", BYTEORDER_BE);
                         break;
                     }
-
+#else
+                    switch (enc.optenc)
+                    {
+                    case OPTENC_UTF8:
+                        bytes = PyUnicode_AsUTF8String(objCell);
+                        break;
+                    case OPTENC_UTF16:
+                        bytes = PyUnicode_AsUTF16String(objCell);
+                        break;
+                    case OPTENC_UTF16LE:
+                        bytes = PyUnicode_AsEncodedString(objCell, "utf_16_le", NULL);
```

Comment:
Minor, but it might be nice to specify the errors parameter as "strict" rather than leaving it as NULL.  Ditto for OPTENC_UTF16BE.

Reply:
In Python/C API, `errors=NULL` means "strict" consistently, like `errors=None` in Python.
And it is little faster/efficient because Python can skip `strcmp()`, creating temporary Unicode object, or looking up error handler from registry.

Do you prefer not using the default anyway?